front/lib/resources/storage/models/runs.ts

```diff
@@ -102,6 +104,11 @@ RunUsageModel.init(
       defaultValue: null,
       allowNull: true,
     },
+    costUsd: {
+      type: DataTypes.FLOAT,
```

Comment:
Finally going to float here to not lose precision, as runs can easily be below 1 cent